api/v1alpha1/database_webhook.go

```diff
@@ -211,7 +212,7 @@ func instanceSpecValidatorForCreate(instance *Instance, allErrs field.ErrorList,
 }
 
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
-func (r *Database) ValidateCreate() error {
+func (r *Database) ValidateCreate() (admission.Warnings, error) {
```

Comment:
This change was needed to upgrade to controller-runtime v0.15.1
Ref: https://book.kubebuilder.io/cronjob-tutorial/webhook-implementation 

Error received without this change:
```
# github.com/nutanix-cloud-native/ndb-operator/api/v1alpha1
api/v1alpha1/database_webhook.go:120:27: cannot use &Database{} (value of type *Database) as admission.Validator value in variable declaration: *Database does not implement admission.Validator (wrong type for method ValidateCreate)
                have ValidateCreate() error
                want ValidateCreate() (admission.Warnings, error)

```